microapps/MapInsetsApp/app/src/main/java/com/arcgismaps/toolkit/mapinsetsapp/screens/MainScreen.kt

```diff
@@ -0,0 +1,140 @@
+            var topText by remember { mutableStateOf(TextFieldValue("")) }
+            var bottomText by remember { mutableStateOf(TextFieldValue("")) }
+
+            val updateInsets = {
+                mapInsets = PaddingValues(
+                    leftText.text.toDoubleOrNull()?.dp ?: 0.0.dp,
+                    topText.text.toDoubleOrNull()?.dp ?: 0.0.dp,
+                    rightText.text.toDoubleOrNull()?.dp ?: 0.0.dp,
+                    bottomText.text.toDoubleOrNull()?.dp ?: 0.0.dp
+                )
+            }
+
+            val focusManager = LocalFocusManager.current
+            MapView(
+                modifier = Modifier.fillMaxSize(),
+                arcGISMap = arcGISMap,
+                mapInsets = mapInsets
+            )
+
+            Column(Modifier.fillMaxSize()) {
+                Row(Modifier.fillMaxWidth(), horizontalArrangement = Arrangement.SpaceEvenly) {
+                    OutlinedTextField(
+                        value = leftText,
+                        modifier = Modifier.size(90.dp, 64.dp),
+                        onValueChange = { text: TextFieldValue -> leftText = text },
+                        label = { Text("Left") },
+                        keyboardOptions = KeyboardOptions(
+                            keyboardType = KeyboardType.Decimal,
+                            imeAction = ImeAction.Done
+                        ),
+                        keyboardActions = KeyboardActions(onDone = { updateInsets(); focusManager.clearFocus() })
+                    )
+                    OutlinedTextField(
+                        value = rightText,
+                        modifier = Modifier.size(90.dp, 64.dp),
+                        onValueChange = { text: TextFieldValue -> rightText = text },
+                        label = { Text("Right") },
+                        keyboardOptions = KeyboardOptions(
+                            keyboardType = KeyboardType.Decimal,
+                            imeAction = ImeAction.Done
+                        ),
+                        keyboardActions = KeyboardActions(onDone = { updateInsets(); focusManager.clearFocus() })
+                    )
+                    OutlinedTextField(
+                        value = topText,
+                        modifier = Modifier.size(90.dp, 64.dp),
+                        onValueChange = { text: TextFieldValue -> topText = text },
+                        label = { Text("Top") },
+                        keyboardOptions = KeyboardOptions(
+                            keyboardType = KeyboardType.Decimal,
+                            imeAction = ImeAction.Done
+                        ),
+                        keyboardActions = KeyboardActions(onDone = { updateInsets(); focusManager.clearFocus() })
+                    )
+                    OutlinedTextField(
+                        value = bottomText,
+                        modifier = Modifier.size(96.dp, 64.dp),
```

Comment:
Should this also be 90dp?
```suggestion
                        modifier = Modifier.size(90.dp, 64.dp),
```

Reply:
I reduced the size of others to 90.dp to make room for this one which has longer text that was getting wrapped. I test it again and it seems to work for 90.dp as well. So I have changed it to make them all the same.